Sources/LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -31,14 +31,25 @@ public enum LaunchAtLogin {
 			}
 
 			kvo.willChangeValue(for: \.isEnabled)
-			SMLoginItemSetEnabled(id as CFString, newValue)
+			Self._SMLoginItemSetEnabled(id: id, isEnabled: newValue)
 			kvo.didChangeValue(for: \.isEnabled)
 
 			if #available(macOS 10.15, *) {
 				_publisher.send(newValue)
 			}
 		}
 	}
+	
+	private static func _SMLoginItemSetEnabled(id: String, isEnabled: Bool) {
```

Comment:
This should be wrapped in an `#if` check for Catalyst. I would also move it to a private top-level function.